.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "OpenWPM"]
 	path = OpenWPM
-	url = https://github.com/englehardt/OpenWPM.git
-[submodule "analysis/crawl_utils"]
-	path = analysis/crawl_utils
-	url = https://github.com/englehardt/crawl_utils.git
+	url = https://github.com/mozilla/OpenWPM.git
+	branch = openwpm-crawler-support
+[submodule "analysis/utils"]
```

Comment:
This can just be removed. The library is now published on pypi (https://pypi.org/project/openwpm-utils/)

Reply:
@motin I'm happy to do this in a future PR